crypto-core/src/main/java/com/palantir/crypto/cipher/AesCtrCipher.java

```diff
@@ -37,11 +35,11 @@
 public final class AesCtrCipher implements SeekableCipher {
 
     public static final String ALGORITHM = "AES/CTR/NoPadding";
-    static final String PROVIDER = "SunJCE";
     static final String KEY_ALGORITHM = "AES";
     static final int KEY_SIZE = 256;
     static final int BLOCK_SIZE = 16;
     static final int IV_SIZE = 16;
+    static byte[] out = new byte[BLOCK_SIZE];
```

Comment:
As far as I can tell it's needed for the call to `cipher.update(skip, 0 /* inputOffset */, bytesToSkip, out, 0 /* outputOffset */);` below. As you say we don't use `out`, so it'd be good if we could find a way to not initialize it, but my reading of https://github.com/apache/commons-crypto/blob/master/src/main/java/org/apache/commons/crypto/cipher/CryptoCipher.java is that we need to write to something for `update` to work.